lib/registration.pm

```diff
@@ -29,7 +29,8 @@ our @EXPORT = qw(
   registration_bootloader_params
   yast_scc_registration
   skip_registration
-  SLE15_MODULES
+  %SLE15_MODULES
+  %SLE15_DEFAULT_MODULES
```

Comment:
@yxususe , @rwx788 I don't know why your previous changes worked but do you think this makes problems for suseconnect now?

Reply:
Hmm, I guess it's correct way, as you've done it. But it works with both ways.

Reply:
Yes, I can confirm both way worked.